### PR TITLE
Test adding and removing bundles

### PR DESCRIPTION
**Pull request by jsteiner**

### CODE REVIEW

File: lib/cog_api/client.ex [hunk 1]
```diff
@@ -46,6 +46,8 @@ defmodule CogApi.Client do
   @callback relay_group_delete(String.t, %Endpoint{}) :: {atom, [String.t]}
   @callback relay_group_add_relay(String.t, String.t, %Endpoint{}) :: {atom, %RelayGroup{}}
   @callback relay_group_remove_relay(String.t, String.t, %Endpoint{}) :: {atom, %RelayGroup{}}
+  @callback relay_group_add_bundle(String.t, String.t, %Endpoint{}) :: {atom, %RelayGroup{}}
```
Comment (drapergeek) [inline lib/cog_api/client.ex]: There is also an `add_bundle` that accepts a `%{name: bundle_name}` on the `HTTP` side.

Comment (drapergeek) [inline lib/cog_api/client.ex]: Probably need to add that to the tests as well as the fake.

Comment (jsteiner) [inline lib/cog_api/client.ex]: I was having some issues with testing that, so I'd rather hold off for now. I don't _think_ we're going need that in Flywheel.

Comment (drapergeek) [inline lib/cog_api/client.ex]: Can you explain more about your issues?

I agree that we're probably not going to need it but also feel really weird about leaving in code that is untested and also something that is not mirrored in the `fake`.

Comment (drapergeek) [inline lib/cog_api/client.ex]: Should there be a line here for the `add_bundle` that takes a map?

**Comment by drapergeek** [comment]
LGTM

**Pull request merged**